data/syndicate world jobs.txt

```diff
@@ -323,9 +326,9 @@ mission "Syndicate target practice [3]"
 		distance 1 3
 		government Syndicate
 	to offer
-		random < 20
+		random < 100
```

Comment:
Why even include the condition?

(unless you `oops'd` and forgot to drop it back after testing :innocent: )